### PR TITLE
`Programming exercises`: Remove mention of SourceTree for cloning

### PR DESCRIPTION
**Pull request by JohannesStoehr**
#### General

- [x] This is a small issue that I tested locally and was confirmed by another developer on a test server.
- [x] Language: I followed the [guidelines for inclusive, diversity-sensitive, and appreciative language](https://docs.artemis.cit.tum.de/dev/guidelines/language-guidelines/).
- [x] I chose a title conforming to the [naming conventions for pull requests](https://artemis-platform.readthedocs.io/en/latest/dev/guidelines/development-process.html#naming-conventions-for-github-pull-requests).
#### Client
- [x] I followed the [coding and design guidelines](https://docs.artemis.cit.tum.de/dev/guidelines/client/) and ensured that the layout is responsive.
- [x] I added multiple screenshots/screencasts of my UI changes.
- [x] I translated all newly inserted strings into English and German.

### Motivation and Context

The tutorial is currently switched from SourceTree to only using IntelliJ for cloning (#5988), so the clone dialog should not put such a big emphasis on using source tree, let alone make an advertisement for this specific git client.

### Description

This PR makes the button label more generic and adds a small help icon, explaining the buttons usage.
The direct link to SourceTree below the button is removed.

### Steps for Testing

Prerequisites:
- 1 Instructor
- 1 Student
- 1 Programming Exercise

1. Log in to Artemis
2. Navigate to Course Administration
3. Try to clone the exercise using the "Clone in git client" button in these cases:
- Without any git client installed (button should do nothing)
- With either SourceTree or Tower installed (button should open the respective git client)(Not every tester has to check every git client)

#### Code Review
- [x] Review 1
- [x] Review 2
#### Manual Tests
- [x] Test 1
- [x] Test 2

### Screenshots

Bildschirm­foto 2023-01-13 um 16 59 49

### CODE REVIEW

**Comment by Strohgelaender** [comment]
Does this actually work with any "git client like Source Tree, Tower or GitKraken"? The URL still says `sourcetree://` at the beginning.

**Review by ge65cer — COMMENTED** [review]
We no longer mention Sourcetree but still specify it in the link. For [GitKraken](https://help.gitkraken.com/gitkraken-client/linking/) as an example the deep link looks different. I don't think there is a generic link, so maybe we need to remove the button completely?

**Review by LiaLism — DISMISSED** [review]
The SourceTree button is removed and the help icon is displayed. The new button works as expected. Tested on ts1.

**Comment by JohannesStoehr** [comment]
> We no longer mention Sourcetree but still specify it in the link. For [GitKraken](https://help.gitkraken.com/gitkraken-client/linking/) as an example the deep link looks different. I don't think there is a generic link, so maybe we need to remove the button completely?

Sadly it does not work for GitKraken, but Tower is finde with that link. I adapted the tooltip to accurately describe what clients are working.

File: src/main/webapp/i18n/de/exercise-actions.json [hunk 1]
```diff
@@ -47,8 +47,10 @@
             "cloneExerciseRepository": "Klone ein Repository für diese Übung",
             "repositoryPassword": "Dein Passwort lautet:",
             "hoverToShow": "(hover, um es anzuzeigen)",
-            "cloneSourceTree": "In SourceTree klonen",
-            "sourceTreeDescription": "<a href='https://www.sourcetreeapp.com' target='_blank'>Atlassian SourceTree</a> ist ein kostenloser GIT-Client für Mac und Windows.",
+            "cloneSourceTree": {
+                "button": "In Git-Client klonen",
+                "helpIcon": "Dies öffnet das Repository in Source Tree oder Tower. Falls du diese nicht installiert hast, kannst du das Repository in deiner IDE oder dem Terminal klonen."
```
Comment (Strohgelaender) [inline src/main/webapp/i18n/de/exercise-actions.json:52]:
```suggestion
                "helpIcon": "Dies öffnet das Repository in SourceTree oder Tower. Falls du diese nicht installiert hast, kannst du das Repository in deiner IDE oder dem Terminal klonen."
```

File: src/main/webapp/i18n/en/exercise-actions.json [hunk 2]
```diff
@@ -48,8 +48,10 @@
             "cloneExerciseRepository": "Clone a repository for this exercise",
             "repositoryPassword": "Your Password is:",
             "hoverToShow": "(hover to show)",
-            "cloneSourceTree": "Clone in SourceTree",
-            "sourceTreeDescription": "<a href='https://www.sourcetreeapp.com' target='_blank'>Atlassian SourceTree</a> is a free Git client for Windows or Mac.",
+            "cloneSourceTree": {
+                "button": "Clone in Git client",
+                "helpIcon": "This opens the repository in Source Tree or Tower. If you do not have them, you can still clone the repository using your IDE or terminal."
```
Comment (Strohgelaender) [inline src/main/webapp/i18n/en/exercise-actions.json:53]:
```suggestion
                "helpIcon": "This opens the repository in SourceTree or Tower. If you do not have them, you can still clone the repository using your IDE or terminal."
```

File: src/main/webapp/i18n/de/exercise-actions.json (hunk 1 above)
Comment (ge65cer) [reply src/main/webapp/i18n/de/exercise-actions.json:52]: Official spelling is `Sourcetree` 😉

**Review by LiaLism — DISMISSED** [review]
Reapproved. Everything works as expected. Tested on ts1.

File: src/main/webapp/i18n/de/exercise-actions.json (hunk 1 above)
Comment (JohannesStoehr) [reply src/main/webapp/i18n/de/exercise-actions.json:52]:
```suggestion
                "helpIcon": "Dies öffnet das Repository in Sourcetree oder Tower. Falls du diese nicht installiert hast, kannst du das Repository in deiner IDE oder dem Terminal klonen."
```

File: src/main/webapp/i18n/en/exercise-actions.json (hunk 2 above)
Comment (JohannesStoehr) [reply src/main/webapp/i18n/en/exercise-actions.json:53]:
```suggestion
                "helpIcon": "This opens the repository in Sourcetree or Tower. If you do not have them, you can still clone the repository using your IDE or terminal."
```

**Review by simon-hng — APPROVED** [review]
Code

**Review by TheZoker — APPROVED** [review]
Code LGTM 👍

**Review by stefanwaldhauser — APPROVED** [review]

**Review by Strohgelaender — APPROVED** [review]

**Pull request merged**